wizard/src/utils/detect_place.test.tsx

```diff
@@ -13,14 +13,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { DetectedFormat } from "../types";
+import { ConfidenceLevel, DetectedDetails, DetectedFormat } from "../types";
 import { PlaceDetector } from "./detect_place";
 
+// Helper function to determine if two DetectedDetails objects are equal.
```

Comment:
there's a lodash function that can be used to accomplish this:  https://lodash.com/docs/4.17.15#isEqual

Reply:
Oh nice. thanks!